ReactiveObjC/RACTuple.h

```diff
@@ -111,15 +115,84 @@ NS_ASSUME_NONNULL_BEGIN
 - (nullable id)objectAtIndexedSubscript:(NSUInteger)idx;
 @end
 
+/// A tuple with exactly two generic values.
+@interface RACTwoTuple<__covariant First, __covariant Second> : RACTuple
```

Comment:
This seems reasonable—I'll go ahead and add `RACOneTuple` with one generic element. However, I'm still not convinced of the benefit of having `RACZeroTuple`—there's no compiler guarantees that come from this. The only benefit is a runtime guarantee that you have zero elements, which consumers can do by asserting on `tuple.count` if desired.